config/locales/en.yml

```diff
@@ -460,6 +461,7 @@ en:
           advocate_category: 'Advocate category'
           case_offence: 'Case & offence'
           case_number: 'Case number'
+          providers_ref: 'Provider reference number'
```

Comment:
Also, we could use a 'common.external_user. providers_ref' key to not duplicate the same string.